install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -1718,17 +1718,6 @@ dashboards:
   labelValue: "1"
   annotations: {}
 
-# -- Configure Istio proxy options.
-proxy:
-
-  prometheus:
-    # -- Deprecated in favor of envoy.prometheus.enabled
-    enabled: true
-    # -- Deprecated in favor of envoy.prometheus.port
-    port: ~
-  # -- Regular expression matching compatible Istio sidecar istio-proxy
-  # container image names
```

Comment:
This one seems to be still required, the comment `Configure Istio proxy options` might be mis-leanding.